main.py

```diff
@@ -1,63 +1,90 @@
-from core.loader import load_train_dataset
+import sys
+import time
+
+from core.loader import load_train_dataset, load_standardized_train_dataset, load_test_dataset
 from core.processing import expand_dataset, split_training_test, split_claims_accept_reject, separate_features_label
-from core.data_visualization import handle_basic_plots, handle_compound_plots
-from core.constants import DATASET_LABEL_NAME, DATASET_TRAIN_RATIO, POLYNOMIAL_ANALYSIS_DEGREES, CROSS_VALIDATION_SETS
-from core.data_analysis import calculate_linear_regression_error, calculate_polynomial_complexity_errors
+from core.data_visualization import generate_data_visualization_plots
+from core.constants import DATASET_LABEL_NAME, DATASET_TRAIN_RATIO, MENU_EXIT, MENU_RETURN
+from core.data_analysis import perform_linear_regression_analysis, perform_polynomial_complexity_analysis, \
+    perform_lasso_lambda_analysis, perform_ridge_lambda_analysis, perform_feature_correlation_analysis
```

Comment:
fwiw i am separating out the decision tree part into its own `main_induction.py` that can be run separately from the main driver and also exposes the perform component for use inside the main menu.. I think that would be more scalable than cramming everything into data_analysis but its your call

Reply:
That sounds good to me.